src/main/java/de/tum/in/www1/artemis/service/plagiarism/PlagiarismService.java

```diff
@@ -75,4 +90,56 @@ else if (plagiarismStatus.equals(PlagiarismStatus.DENIED)) {
             plagiarismCaseService.removeSubmissionsInPlagiarismCasesForComparison(plagiarismComparisonId);
         }
     }
+
+    /**
+     * Retrieves the number of potential plagiarism cases by considering the plagiarism submissions for the exercise
+     * Additionally, it filters out cases for deleted user --> isDeleted = true because we do not delete the user entity entirely.
+     *
+     * @param exerciseId the exercise id for which the potential plagiarism cases should be retrieved
+     * @return the number of potential plagiarism cases
+     */
+    public long getNumberOfPotentialPlagiarismCasesForExercise(long exerciseId) {
+        var comparisons = plagiarismComparisonRepository.findAllByExerciseId(exerciseId);
+        Set<PlagiarismSubmission<?>> submissionsWithoutDeletedUsers = new HashSet<>();
+        for (var comparison : comparisons) {
+            addSubmissionsIfUserHasNotBeenDeleted(comparison, submissionsWithoutDeletedUsers);
+        }
+        return submissionsWithoutDeletedUsers.size();
+    }
+
+    /**
+     * Add each submission of the plagiarism comparison if the corresponding user has not been deleted
+     *
+     * @param comparison                     the comparison for which we want check if the user of the submission has been deleted.
```

Comment:
The formatting of this JavaDoc is inconsistent with the other ones.

Reply:
This is expected and automatically done by spotless. The descriptions for `@param` tags all start at the same level, the long whitespace is due to the long name of `submissionsWithoutDeletedUsers` below.